css/screens/landingPage.css

```diff
@@ -0,0 +1,136 @@
+/*  ==== css for landing page === */
+
+.big-wrapper{
+    padding: 1.7rem 0 2rem;
+    width: 100%;
+    min-height: 100vh;
+    overflow: hidden;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+.container{
+    max-width: 81rem;
+    width: 100%;
+    margin: 0 auto;
+    padding: 0 3rem;
+}
+header .container{
+    position: relative;
+    z-index: 10;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.logo{
+    display: flex;
+    align-items: center;
+}
+.logo>h3{
+    font-size: var(--h3-font-size);
+    font-weight: var(--font-semi-bold);
+}
+.logo img{
+    width: 50px;
+    margin-right: .6rem;
+}
+.links >ul{
+    display: flex;
+    align-items: center;
+}
+.links a{
+    color: var(--text-color) ;
+    margin-left: 4.5rem;
+    display: inline-block;
+    transition: .1s;
+}
+.links a:hover{
+    color: var(--primary-dark);
+    transform: scale(1.05);
+}
+.btn{
+    display: inline-block;
+    padding: .9rem 1.9rem;
+    color: #fff !important;
+    border-radius: .5rem;
+    box-shadow: var(--box-shadow);
+    background-color: var(--primary);
+    transition: .3s;
+}
+.btn:hover{
+    background-color: var(--primary-dark);
+    transform: scale(1) !important;
+}
+
+.showcase-area .container{
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+    justify-content: center;
+    align-items: center;
+}
+.big-title{
+    font-size: var(--h2-font-size);
+    color: var(--text-color);
+    text-transform: capitalize;
+    line-height: 1.4;
+}
+.text{
+    color: var(--text-color-light);
+    font-size: var(--normal-font-size);
+    margin: 1.9rem 0 2rem;
+    max-width: 550px;
+    line-height: 1.7;
+}
+/* .right-side{
+    padding: 0rem 1rem;
+} */
```

Comment:
do remove these unnecessary blocks of comments

Reply:
sure will remove that